HPXMLtoOpenStudio/resources/geometry.rb

```diff
@@ -424,6 +424,14 @@ def self.get_temperature_scheduled_space_values(location)
                outdoor_weight: 0.0,
                ground_weight: 1.0,
                f_regain: 0.83 } # From LBNL's "Technical Background for default values used for Forced Air Systems in Proposed ASHRAE Standard 152P"
+    elsif location == HPXML::LocationManufacturedHomeBelly
+      # From LBNL's "Technical Background for default values used for Forced Air Systems in Proposed ASHRAE Standard 152P"
+      # 3.5 Manufactured House Belly Pan Temperatures
```

Comment:
```suggestion
      # From LBNL's "Technical Background for default values used for Forced Air Systems in Proposed ASHRAE Standard 152P"
      # 3.5 Manufactured House Belly Pan Temperatures
      # FUTURE: Consider modeling the belly as a separate thermal zone so that we dynamically calculate temperatures.
```